WalletWasabi.Fluent/Views/Wallets/Send/TransactionSummary.axaml

```diff
@@ -72,7 +72,7 @@
         </c:PreviewItem>
         <c:PreviewItem Icon="{StaticResource paper_cash_regular}"
                        Label="there will be an additional transaction fee of">
-          <TextBlock Text="{Binding FeeText, FallbackValue=0.00000235 bitcoin (≈55.34 USD)}" />
+          <TextBlock Classes="monoSpaced" Text="{Binding FeeText, FallbackValue=0.0000 0235 BTC (≈55.34 USD)}" />
```

Comment:
What does it mean fallback value? In case of a bug when FeeText is null or empty, we will display false information?

Reply:
I believe this is for previewer purposes only, the amounts will never be null unless it's missing a datacontext, in which case this fallback value will show up. And that only happens in previewer in Rider/VS

Reply:
Isn't there any 'designer' property like in WPF? The problem with this, it seems like real values, and there is no chance to see if we are in the fallback scenario. Can you put something there indicating the error in case this gets in front of the user, like 9.999999 BTC or something.

Reply:
Yes that makes sense, will add that now